src/util.h

```diff
@@ -68,23 +70,18 @@ inline std::string _(const char* psz)
 void SetupEnvironment();
 bool SetupNetworking();
 
-/** Return true if log accepts specified category */
-bool LogAcceptCategory(const char* category);
 /** Send a string to the log output */
 int LogPrintStr(const std::string &str);
 
-#define LogPrintf(...) LogPrint(NULL, __VA_ARGS__)
```

Comment:
In the new code, `LogPrintf()` (unconditional logging statement) isn't defined as `LogPrint()`, because doing so would cause `LogPrintf()` to statically allocate a `LogFastEntry` structure, which would be unnecessary because these statements are always enabled. So instead, in the new code, `LogPrintf()` is defined to just call `LogPrintStr()` (which does the actual log printing work).